hack/verify-generated-docs.sh

```diff
@@ -16,6 +16,10 @@ GENERATED_DOCS_ROOT="${OS_ROOT}/${GENERATED_DOCS_ROOT_REL}"
 TMP_GENERATED_DOCS_ROOT_REL="_output/verify-generated-docs"
```

Comment:
These tasks need to be added to `make verify` and `make update` or they won't be run.